tower-layer/Cargo.toml

```diff
@@ -6,7 +6,7 @@ name = "tower-layer"
 #   - README.md
 # - Update CHANGELOG.md.
 # - Create "v0.1.x" git tag.
-version = "0.3.1"
+version = "0.3.2"
```

Comment:
Please split these changes into a separate PR, also how?! 😅 did someone publish a new release and forget to push to GitHub?

Reply:
> Please split these changes into a separate PR, also how?!

Done! https://github.com/tower-rs/tower/pull/729

> did someone publish a new release and forget to push to GitHub?

and no idea haha